hr_attendance_modification_tracking/models/hr_attendance.py

```diff
@@ -17,3 +19,28 @@ class HrAttendance(models.Model):
     check_out = fields.Datetime(
         track_visibility='onchange'
     )
+    time_changed_manually = fields.Boolean(
+        string="Time changed",
+        required=False,
+        compute="_compute_time_changed_manually",
+        default=False,
+        store=True
+    )
+
+    @api.one
```

Comment:
Travis is complaining that `api.one` is deprecated. Better use `for record in self`

Reply:
Noted. Travis was giving error because api.one.
Thanks.